rllib/algorithms/ppo/ppo_tf_policy.py

```diff
@@ -44,6 +44,17 @@ def validate_config(config: AlgorithmConfigDict) -> None:
         )
 
 
+def warn_kl_loss(mean_kl_loss):
```

Comment:
I think we want to continue logging at each failure. A policy can recover from a single bad update, but after a few bad updates it usually cannot.

Reply:
If the user is seeing this fill their logs, their policy is 100% crap. Maybe we should just raise an error instead, idk. I'm also worried if we log once halfway through training, it will be lost in all the metrics/other stuff that RLlib spews out.

Is there a library function for log_every_n_seconds?

Reply:
min_time_iter_s controls log frequency over time. 

Might be wrong about the exact name

There's also a flag for timesteps trained and timesteps sampled.

Reply:
We can also log once.

Reply:
I'm kinda in agreement in Steven though. We should kill runs where the kl blows up. 

Those are probably useless

Reply:
ok, how about we meet in the middle and just flood the output log with this warning?
let's fix lint and get this merged?